ui/src/testing/react-test-helpers.tsx

```diff
@@ -204,6 +204,13 @@ export const expectDropdownDisabled = (
   ).not.toBeInTheDocument();
 };
 
+export const getDropdownSelection = (
```

Comment:
My guess is that this is difficult because it's PrimeReact?

But I have a different comment: we have `id` for these elements that can be used here.  Is it necessary to duplicate this with `name` ?

Reply:
The prime react dropdown has select in the following way beause of which i have to use name
eg
<select
                  name="runtime-cpu"
                  tabindex="-1"
                  aria-hidden="true"
                  data-pc-section="select"
                >